src/snowballs/SnowballDispenser.java

```diff
@@ -0,0 +1,81 @@
+package snowballs;
+
+public class SnowballDispenser {
+
+    private SnowballTarget target;
+    private SnowballListener listener;
+    private SnowballSpeed speed = SnowballSpeed.SLOW;
```

Comment:
Declaration of speed with value
Requiring speed value in Constructor may be a favorable solution.

Compare to GameScene line 46 where a speed is also set